samples/webview-samples/multi-tab-browser/main.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Listens for the app launching then creates the window
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ * @see http://developer.chrome.com/apps/app.window.html
+ */
+window.chrome.app.runtime.onLaunched.addListener(function() {
+  runApp();
+});
+
+/**
+ * Listens for the app restarting then re-creates the window.
+ *
+ * @see http://developer.chrome.com/apps/app.runtime.html
+ */
+window.chrome.app.runtime.onRestarted.addListener(function() {
```

Comment:
We usually don't use `window` as a prefix and simply use `chrome.app.runtime...`